backend/src/main/java/me/quizzl/backend/models/Quiz.java

```diff
@@ -24,8 +22,50 @@
      * This value is automatically generated when a quiz entity is saved to the database.
      */
     private UUID id;
+    private String quizTitle;
 
+    @ElementCollection
```

Comment:
`@ElementCollection` seems to be meant for collections of non-entites, I think `@OneToMany` or `@ManyToMany` might work better here, depending on what we want. If we might want users to create questions that can be used across different quizzes, we'd probably just want `@ManyToMany`

Reply:
I just couldn't let go of @ElementCollection. I removed it in the latest commits and replaced it with @OneToMany for simplicity's sake for the time being but we can always update and move to @ManyToMany in the future if that turns out to be more useful.